test/cypress/e2e/send.spec.js

```diff
@@ -162,29 +162,44 @@ describe('Send', () => {
   });
 
   /**
-   * Initialization dialogue and functionality is shown for the fresh account
+   * Initialization dialogue and functionality shouldn't be shown through wallet send.
+   * @expect initialization dialogue is shown
    * @expect successfully go through initialization
    * @expect transfer transaction appear with correct data
    * @expect initialization dialogue is not shown anymore
    */
-  it('Should be able to init account when needed', () => {
+  it('Should show initialize banner with account not initialized', () => {
     cy.autologin(accounts['without initialization'].passphrase, networks.devnet.node);
     cy.reload();
-    cy.visit(urls.send);
+    cy.visit(urls.dashboard);
+    cy.get(ss.initializeBanner).should('exist');
+    cy.get(ss.initializeBanner).find('a').click();
+    cy.get(ss.backButton).click();
     cy.get(ss.accountInitializationMsg).get(ss.accountInitializationBtn).click();
     cy.get(ss.sendBtn).click();
     cy.get(ss.resultMessage).should('have.text', msg.transferTxSuccess);
     cy.get(ss.accountInitializationMsg).should('not.exist');
     cy.wait(txConfirmationTimeout);
     cy.reload();
-    cy.visit(urls.send);
-    cy.get(ss.accountInitializationMsg).should('not.exist');
     cy.visit(urls.wallet);
     cy.get(ss.transactionRow).eq(0).as('tx');
     cy.get('@tx').find(ss.transactionAddress).should('have.text', accounts['without initialization'].address);
     cy.get('@tx').find(ss.transactionReference).should('have.text', 'Account initialization');
   });
 
+  /**
+   * Initialization dialogue and functionality shouldn't be shown through wallet send.
+   * @expect initialization dialogue is not shown
+   */
+  it('Should be able to send even with account not initialized', () => {
+    cy.autologin(accounts['without initialization'].passphrase, networks.devnet.node);
+    cy.reload();
+    cy.visit(urls.wallet);
+    cy.get(ss.transactionSendButton).click();
+    cy.get(ss.accountInitializationMsg).should('not.exist');
+    cy.get(ss.recipientInput).should('exist');
+  });
+
   /**
```

Comment:
Here you use the 'without init' account, which has already been initialized in the previous test. The blockchain snapshot is reapplied only before the start of the tests, so you shouldn't use this acc anymore or implement test somehow dependent on other tests. As this one is a negative check and you will need another not initialized account I would suggest to include the check into the previous existing test